src/components/playback/playbackmanager.js

```diff
@@ -72,7 +72,6 @@ define(['events', 'datetime', 'appSettings', 'itemHelper', 'pluginManager', 'pla
         const now = (new Date).getTime();
 
         if (method !== reportPlaybackLastMethod || now - (reportPlaybackLastTime || 0) >= reportPlaybackLogDelay) {
-            console.debug(method + '-' + JSON.stringify(info));
```

Comment:
This block is for logging. It was more verbose :p
https://github.com/jellyfin/jellyfin-web/commit/bffafa69143b110f4a550c66642a4497366fad5b

Reply:
Ah, well I still think it's way too verbose for the debug logs :sweat_smile: might just want to remove it.